web-ui/src/main/resources/catalog/views/default/module.js

```diff
@@ -147,6 +147,7 @@
       $scope.modelOptions = angular.copy(gnGlobalSettings.modelOptions);
       $scope.modelOptionsForm = angular.copy(gnGlobalSettings.modelOptions);
       $scope.isFilterTagsDisplayedInSearch = gnGlobalSettings.gnCfg.mods.search.isFilterTagsDisplayedInSearch;
+      $scope.showMapInFacet = gnGlobalSettings.gnCfg.mods.search.showMapInFacet;
       $scope.showStatusFooterFor = gnGlobalSettings.gnCfg.mods.search.showStatusFooterFor;
       $scope.exactMatchToggle = gnGlobalSettings.gnCfg.mods.search.exactMatchToggle;
```

Comment:
As mentioned, I guess something is missing around here:
https://github.com/georchestra/geonetwork/blob/georchestra-gn4-4.x/web/src/main/webapp/WEB-INF/classes/setup/sql/data/data-db-default.sql#L570-L707

But I don't know the mechanisms translating the keys from the settings table to JS variables, to be honest.

What sould be added (if any) into the table ?